grafana/overlays/moc/smaug/grafana-oauth.yaml

```diff
@@ -19,7 +19,8 @@ spec:
       root_url: https://grafana.operate-first.cloud
     dashboards:
       default_home_dashboard_path: "/etc/grafana-configmaps/grafana-landing-page/landingpage.json"
-    serviceAccount:
+  serviceAccount:
```

Comment:
indentation is broken , bring it out of the config
you can keep it above config.